pipelines/bootstrap.sh

```diff
@@ -163,16 +163,34 @@ function clean_openshift_pipelines() {
 
 }
 
-export BASEDIR=$(dirname "$0")
+
+if [[ "${0}" == *"bootstrap.sh" ]]; then
```

Comment:
I think you want to use something like: 

`SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )`

`${0}` is not guaranteed to be `bootstrap.sh` if you run `bash bootstrap.sh` or in some other form